### PR TITLE
[US-3415] CHANGELOG new format

### PR DESCRIPTION
**Pull request by MattCzerner**
| Q             | A
| ------------- | ---
|Description, reason for the PR| new format of changelog for better maintainability
|New feature| Yes
|BC breaks| Yes
|Fixes issues| ...
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

CHANGELOG.md now contains changelogs of all the repositories and got new format with links to the pull requests.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,28 +23,912 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+###### Changed
+- friendsofphp/php-cs-fixer upgraded from version 2.1 to version 2.3 
+- [phpcs-fixer ruleset](./build/phpcs-fixer.php_cs) 
+    - replaced deprecated "hash_to_slash_comment" rule with "single_line_comment_style" rule
+    - custom NoUnusedImportsFixer replaced with standard "no_unused_imports" rule
+
+###### Deleted
+- Redundant rules which were already covered by other rules
+
+### [shopsys/form-types-bundle]
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: Please remove this subtitle as it doesn't contain any content

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,28 +23,912 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+    - see https://github.com/symfony/swiftmailer-bundle/commit/5edfbd39eaefb176922a346c16b0ae3aaeec87e0
+    - the new setting requires array instead of string so the parameter `mailer_master_email_address` is wrapped into array in config
+- [REVISION(fbadde0966e92941dd470591d6a8a4924a798aa8)](https://github.com/shopsys/shopsys/commit/fbadde0966e92941dd470591d6a8a4924a798aa8): `FpJsFormValidator` error in console on FE order pages
+- [PR(#110)](https://github.com/shopsys/shopsys/pull/110): failure during Docker image build triggered by `E: Unable to locate package postgresql-client-9.5`
+
+#### Removed
+- [PR(#94)](https://github.com/shopsys/shopsys/pull/94): support of installation using Docker for Windows 10 Home and lower
+    - virtualization is extremely slow, native installation has much better results in such case
+
+### 7.0.0-alpha1 - 2018-04-12
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: This subtitle should be 2. level

File: UPGRADE.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+#### From 0.1.0 to 0.2.0
+- Rename all implementations and usages of `FeedItemInterface::getItemId()` to `getId()`.
+- Rename all implementations and usages of `FeedItemCustomValuesProviderInterface` to `HeurekaCategoryNameProviderInterface`.
+- If you are using custom values in your implementation, you need to implement interfaces from package [shopsys/plugin-interface](https://github.com/shopsys/plugin-interface) (see [how to work with data storage interface](https://github.com/shopsys/plugin-interface#data-storage)).
+
+### [shopsys/plugin-interface]
+#### From 0.2.0 to 0.3.0
+- all implementations of `DataStorageInterface` now must have implemented method `getAll()` for getting all saved data indexed by keys
+
+### [shopsys-project-base]
```
Comment (simara-svatopluk) [inline UPGRADE.md]: There is missing link for this subtitle

File: UPGRADE.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    (you can run `phing eslint-fix` to fix some violations automatically)  
+
+#### From 2.0.0-beta.15.0 to 2.0.0-beta.16.0  
+- all implementations of `Shopsys\ProductFeed\FeedItemRepositoryInterface` must implement interface `Shopsys\FrameworkBundle\Model\Feed\FeedItemRepositoryInterface` instead  
+    - the interface was moved from [shopsys/product-feed-interface](https://github.com/shopsys/product-feed-interface/) to core  
+- parameter `email_for_error_reporting` was renamed to `error_reporting_email_to` in `app/config/parameter.yml.dist`,  
+    you will be prompted to fill it out again during `composer install`  
+- all implementations of `StandardFeedItemInterface` must implement methods `isSellingDenied()` and `getCurrencyCode()`, see [product-feed-interface](https://github.com/shopsys/product-feed-interface/blob/master/UPGRADE.md#from-030-to-040) 
+
+###[shopsys/coding-standards]
```
Comment (simara-svatopluk) [inline UPGRADE.md]: Formating is malformed because of missing space. Correct is:
`### [shopsys/coding-standards]`

File: UPGRADE.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
+# UPGRADING 
+## [From 7.0.0-alpha1 to Unreleased] 
+### [shopsys/project-base] 
+#### Upgrade instructions
```
Comment (simara-svatopluk) [inline UPGRADE.md]: Personally I would remove this subtitle because all these instructions are "Upgrade instructions"

File: CHANGELOG.md [hunk 6]
```diff
@@ -23,28 +23,912 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+        - native installation is also supported (recommended on Windows 10 Home and lower)
+    - as a rule, using minor versions of docker images (eg. `1.2` or `1.2-alpine`) if possible
+    - home directory of the new user is `/home/www-data/` instead of `/root/`
+    - working directory shortened to the default `/var/www/html` (from `/var/www/shopsys-framework/`)
+    - docs and `docker-compose.yml` templates reflect [changes of Docker images in shopsys/project-base](./project-base/CHANGELOG.md)
+    - `docker-compose-win.yml.dist` created for Windows OS which creates local volume because of permission problems with
+        `postgresql` mounting
+    - docs: changed `./phing` instruction code with `php phing` to make it work with all operating systems
 
 ### Fixed
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: This subtitle should be 4th level

File: CHANGELOG.md [hunk 7]
```diff
@@ -23,28 +23,912 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- documentation was moved into the main [Shopsys repository](https://github.com/shopsys/shopsys/blob/master/docs)
+
+## Before monorepo
+Before we managed to implement monorepo for our packages, we had slightly different versions for each of our package, we had stored our packages on internal server so we dont have PR available.
+That's why is this section formatted differently.
+
+### [shopsys/http-smoke-testing]
+#### [1.1.0](https://github.com/shopsys/http-smoke-testing/compare/v1.0.0...v1.1.0) - 2017-11-01 
+##### Added
+[CONTRIBUTING.md](https://github.com/shopsys/http-smoke-testing/blob/master/CONTRIBUTING.md)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: Please add bullet like:
`- [CONTRI....`

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,50 +1,952 @@
...
+- updated phpunit/phpunit to version 7
+
+#### [0.2.2](https://github.com/shopsys/product-feed-heureka-delivery/compare/v0.2.1...v0.2.2) - 2017-10-04
+##### Added
+- support for [shopsys/product-feed-interface] 0.5.0
+
+#### [0.2.1](https://github.com/shopsys/product-feed-heureka-delivery/compare/v0.2.0...v0.2.1) - 2017-09-25
+##### Added
+- [CONTRIBUTING.md](https://github.com/shopsys/product-feed-heureka-delivery/blob/master/CONTRIBUTING.md)
+- [template for github pull requests](https://github.com/shopsys/product-feed-heureka-delivery/blob/master/docs/PULL_REQUEST_TEMPLATE.md)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,50 +1,952 @@
...
+##### Added
+- This Changelog (@vitek-rostislav)
+- New dependencies for dev(@MattCzerner)
+    - phpunit/phpunit >=5.0.0,<6.0
+    - twig/twig 1.34.0
+    - twig/extensions 1.3.0
+- New automatic test that is controlling right behaviour of plugin
+- Added travis build icon into [README.md](https://github.com/shopsys/product-feed-heureka-delivery/blob/master/README.md)
+##### Changed
+- Dependency [plugin-interface](./shopsys/product-feed-interface) upgraded from ~0.2.0 to ~0.3.0
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,50 +1,952 @@
...
+- `.travis.yml` file with Travis CI configuration
+
+### [shopsys/migrations]
+#### [2.3.0](https://github.com/shopsys/migrations/compare/v2.2.0...v2.3.0 ) - 2018-02-19
+##### Changed
+- services.yml updated to Symfony 3.4 best practices
+
+### [shopsys/product-feed-interface]
+#### [0.5.0](https://github.com/shopsys/product-feed-interface/compare/v0.4.0...v0.5.0) - 2017-10-04
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) contains ID of its main category
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,50 +1,952 @@
...
+
+### [shopsys/product-feed-interface]
+#### [0.5.0](https://github.com/shopsys/product-feed-interface/compare/v0.4.0...v0.5.0) - 2017-10-04
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) contains ID of its main category 
+
+#### [0.4.0](https://github.com/shopsys/product-feed-interface/compare/v0.3.0...v0.4.0) - 2017-09-25
+##### Added
+- [CONTRIBUTING.md](CONTRIBUTING.md) 
+- [template for github pull requests](docs/PULL_REQUEST_TEMPLATE.md) 
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) has new method `isSellingDenied()`
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,50 +1,952 @@
...
+### [shopsys/product-feed-interface]
+#### [0.5.0](https://github.com/shopsys/product-feed-interface/compare/v0.4.0...v0.5.0) - 2017-10-04
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) contains ID of its main category 
+
+#### [0.4.0](https://github.com/shopsys/product-feed-interface/compare/v0.3.0...v0.4.0) - 2017-09-25
+##### Added
+- [CONTRIBUTING.md](CONTRIBUTING.md) 
+- [template for github pull requests](docs/PULL_REQUEST_TEMPLATE.md) 
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) has new method `isSellingDenied()` 
+- [`FeedConfigInterface`](src/FeedConfigInterface.php) has new method `getAdditionalInformation()`
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,50 +1,952 @@
...
+#### [0.5.0](https://github.com/shopsys/product-feed-interface/compare/v0.4.0...v0.5.0) - 2017-10-04
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) contains ID of its main category 
+
+#### [0.4.0](https://github.com/shopsys/product-feed-interface/compare/v0.3.0...v0.4.0) - 2017-09-25
+##### Added
+- [CONTRIBUTING.md](CONTRIBUTING.md) 
+- [template for github pull requests](docs/PULL_REQUEST_TEMPLATE.md) 
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) has new method `isSellingDenied()` 
+- [`FeedConfigInterface`](src/FeedConfigInterface.php) has new method `getAdditionalInformation()` 
+- [`StandardFeedItemInterface`](src/StandardFeedItemInterface.php) has new method `getCurrencyShortcut()`
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,50 +1,952 @@
...
+ - [CONTRIBUTING.md](https://github.com/shopsys/plugin-interface/blob/master/CONTRIBUTING.md)
+ - `DataStorageInterface` can return all saved data via `getAll()`
+ - `IteratedCronModuleInterface` and `SimpleCronModuleInterface`
+ 
+ ## [0.2.0](https://github.com/shopsys/plugin-interface/compare/v0.1.0...v0.2.0) - 2017-09-06
+ ### Added
+ - This Changelog
+ - interface for loading plugin's demo data
+     - `PluginDataFixtureInterface`
+ - [template for github pull requests](https://github.com/shopsys/plugin-interface/blob/master/PULL_REQUEST_TEMPLATE.md)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,50 +1,952 @@
...
+- DatabaseSearching: added getFullTextLikeSearchString() 
+- admin: E-mail newsletter: now contains list of registered e-mails with ability to delete them
+
+##### Changed
+- cache is cleared before PHPUnit tests only when run via [Phing targets](https://github.com/shopsys/shopsys/blob/master/docs/introduction/phing-targets.md), not when run using `phpunit` directly 
+- PHPUnit tests now fail on warning 
+- end of support of PHP 7.0 
+- renamed TermsAndCondition to LegalCondition to avoid multiple classes for legal conditions agreements 
+- emails with empty subject or body are no longer sent
+- postgresql-client is installed in [php-fpm/dockerfile](./docker/php-fpm/Dockerfile) for `pg_dump` function
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,50 +1,952 @@
...
+- released only in closed beta
+
+##### Fixed
+- coding standards check "phing standards" passes
+
+#### 6.0.0-beta19 - 2017-11-21
+- released only in closed beta
+
+##### Added
+- size of performance data fixtures and limits for performance testing are now configurable via parameters defined in [`parameters_common.yml`](./app/config/parameters_common.yml)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,50 +1,952 @@
...
+- dependency [shopsys/product-feed-heureka-delivery](https://github.com/shopsys/product-feed-heureka-delivery/) upgraded from 0.1.1 to 0.2.0
+- dependency [shopsys/product-feed-interface](https://github.com/shopsys/product-feed-interface/) upgraded from 0.2.1 to 0.3.0
+- it is no longer needed to redeclare feed plugin's implementations of `FeedConfigInterface` in `services.yml`
+    - decision about providing proper instance of `FeedItemRepositoryInterface` is made in `FeedConfigFacade`
+- FeedConfigRepository renamed to `FeedConfigRegistry` 
+    - it is not fetching data from Doctrine as other repositories, it only serves as a container for registering services of specific type
+    - similar to `PluginDataFixtureRegistry` or `PluginCrudExtensionRegistry`
+- `UknownPluginDataFixtureException` renamed to `UnknownPluginCrudExtensionTypeException` because of a typo 
+- `FeedConfigRegistry` now contains all FeedConfigs in one array (indexed by type)
+    - definition and assertion of known feed configs types moved from [`RegisterProductFeedConfigsCompilerPass`](./src/Shopsys/ShopBundle/DependencyInjection/Compiler/RegisterProductFeedConfigsCompilerPass.php) to `FeedConfigRegistry`
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,50 +1,952 @@
...
+    - changed message and arguments of `UnknownFeedConfigTypeException`
+- renamed methods working with standard feeds only to be more expressive 
+    - renamed `FeedConfigFacade::getFeedConfigs()` to `getStandardFeedConfigs()`
+    - renamed `FeedFacade::generateFeedsIteratively()` to `generateStandardFeedsIteratively()`
+    - renamed `FeedGenerationConfigFactory::createAll()` to `createAllForStandardFeeds()`
+- [`parameters.yml.dist`](./app/config/parameters.yml.dist): renamed parameter `email_for_error_reporting` to `error_reporting_email_to`
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,50 +1,952 @@
...
+- sender email for error reporting is now configured in [`parameters.yml.dist`](./app/config/parameters.yml.dist)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 20]
```diff
@@ -1,50 +1,952 @@
...
+- reimplemented `CategoriesType` (@Petr Heinz)
+    - it now extends `CollectionType` instead of `ChoiceType`
+    - it loads only those categories that are needed to show all selected categories in a tree, not all of them
+    - collapsed categories can be loaded via AJAX
+- `CategoryRepository::findById()` now uses `find()` method of Doctrine repository instead of query builder so it can use cached results 
+- it is possible to mention occurrences of an image size in [`images.yml`](./src/Shopsys/ShopBundle/Resources/config/images.yml)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 21]
```diff
@@ -1,50 +1,952 @@
...
+    - it now extends `CollectionType` instead of `ChoiceType`
+    - it loads only those categories that are needed to show all selected categories in a tree, not all of them
+    - collapsed categories can be loaded via AJAX
+- `CategoryRepository::findById()` now uses `find()` method of Doctrine repository instead of query builder so it can use cached results 
+- it is possible to mention occurrences of an image size in [`images.yml`](./src/Shopsys/ShopBundle/Resources/config/images.yml) 
+    - previously they were directly in `ImageController`
+    - they are not translatable anymore (too hard to maintain)
+
+##### Removed
+- email for error reporting removed from [`parameters_test.yml.dist`](app/config/parameters_test.yml.dist)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 22]
```diff
@@ -1,50 +1,952 @@
...
+- removed unused private properties from classes 
+- removed `CategoriesTypeTransformerFactory` 
+    - the `CategoriesTypeTransformer` can be fully autowired after deletion of `$domainId`
+
+##### Fixed
+- [`InlineEditPage::createNewRow()`](tests/ShopBundle/Acceptance/acceptance/PageObject/Admin/InlineEditPage.php) now waits for AJAX to complete
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 23]
```diff
@@ -1,50 +1,952 @@
...
+
+##### Removed
+- email for error reporting removed from [`parameters_test.yml.dist`](app/config/parameters_test.yml.dist)
+- removed unused private properties from classes 
+- removed `CategoriesTypeTransformerFactory` 
+    - the `CategoriesTypeTransformer` can be fully autowired after deletion of `$domainId`
+
+##### Fixed
+- [`InlineEditPage::createNewRow()`](tests/ShopBundle/Acceptance/acceptance/PageObject/Admin/InlineEditPage.php) now waits for AJAX to complete 
+    - fixes false negatives of acceptance test [`PromoCodeInlineEditCest::testPromoCodeCreate()`](tests/ShopBundle/Acceptance/acceptance/PromoCodeInlineEditCest.php)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 24]
```diff
@@ -1,50 +1,952 @@
...
+##### [4.0](https://github.com/shopsys/coding-standards/compare/v3.1.1...v4.0.0) - 2018-01-27
+###### Added
+- composer script `run-all-checks` for easier testing of the package (@TomasVotruba)
+
+###### Changed
+- `OrmJoinColumnRequireNullableFixer` marked as *risky* (@sustmi)
+- [#11](https://github.com/shopsys/coding-standards/pull/11) dropped support of PHP 7.0 
+- [#12](https://github.com/shopsys/coding-standards/pull/12/) [EasyCodingStandard](https://github.com/Symplify/EasyCodingStandard) is now used (@TomasVotruba)
+    - the tool encapsulates PHP-CS-Fixer and PHP_CodeSniffer 
+    - rules configuration is now unified in single file - [`easy-coding-standard.neon`](./easy-coding-standard.neon)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 25]
```diff
@@ -1,50 +1,952 @@
...
+- line length sniff (@TomasVotruba)
+
+##### [3.1.1](https://github.com/shopsys/coding-standards/compare/v3.1.0...v3.1.1) - 2017-10-31
+###### Fixed
+- enabled custom fixers
+
+##### [3.1.0](https://github.com/shopsys/coding-standards/compare/v3.0.2...v3.1.0) - 2017-10-12
+###### Added
+- This changelog 
+- [Description of used coding standards rules](./docs/description-of-used-coding-standards-rules.md)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 26]
```diff
@@ -1,50 +1,952 @@
...
+
+##### [3.1.1](https://github.com/shopsys/coding-standards/compare/v3.1.0...v3.1.1) - 2017-10-31
+###### Fixed
+- enabled custom fixers
+
+##### [3.1.0](https://github.com/shopsys/coding-standards/compare/v3.0.2...v3.1.0) - 2017-10-12
+###### Added
+- This changelog 
+- [Description of used coding standards rules](./docs/description-of-used-coding-standards-rules.md) 
+- New rules in [phpcs-fixer ruleset](./build/phpcs-fixer.php_cs):
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

File: CHANGELOG.md [hunk 27]
```diff
@@ -1,50 +1,952 @@
...
+    - phpdoc_single_line_var_spacing
+    - protected_to_private
+    - semicolon_after_instruction
+    - short_scalar_cast
+    - space_after_semicolon
+    - whitespace_after_comma_in_array
+
+###### Changed
+- friendsofphp/php-cs-fixer upgraded from version 2.1 to version 2.3 
+- [phpcs-fixer ruleset](./build/phpcs-fixer.php_cs)
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: 404

**Comment by simara-svatopluk** [comment]
Anyway please tell me - the UPGRADE.md have to be removed or not?

File: CHANGELOG.md (hunk 1 above)
Comment (MattCzerner) [reply CHANGELOG.md]: I dont know what you mean but this section acutally got changes for 2 versions, i think that indention of versions under that was the problem, isnt that right? i fixed it here c96c0b9e1851336e97ee9a236ab6c318de690726

File: CHANGELOG.md (hunk 2 above)
Comment (MattCzerner) [reply CHANGELOG.md]: you are right

File: CHANGELOG.md [hunk 28]
```diff
@@ -1,50 +1,952 @@
...
+
+#### [0.4.1](https://github.com/shopsys/product-feed-heureka/compare/v0.4.0...v0.4.1) - 2017-09-25
+##### Added
+- [CONTRIBUTING.md](https://github.com/shopsys/product-feed-heureka/blob/master/CONTRIBUTING.md) 
+- [template for github pull requests](https://github.com/shopsys/product-feed-heureka/blob/master/docs/PULL_REQUEST_TEMPLATE.md) 
+##### Changed
+- Dependency [product-feed-interface](https://github.com/shopsys/product-feed-heureka/blob/master/shopsys/product-feed-interface) upgraded from ~0.3.0 to ~0.4.0 
+- [`HeurekaFeedConfig`](https://github.com/shopsys/product-feed-heureka/blob/master/src/HeurekaFeedConfig.php) now filters not sellable products 
+- [`HeurekaFeedConfig`](https://github.com/shopsys/product-feed-heureka/blob/master/src/HeurekaFeedConfig.php) implemented method `getAdditionalData()` 
+- [`TestStandardFeedItem`](https://github.com/shopsys/product-feed-heureka/blob/master/tests/TestStandardFeedItem.php) implemented method `getCurrencyCode()`
```
[the remark this answers is not in the document]
Comment (MattCzerner) [reply CHANGELOG.md]: this is historical part of monorepo, i dont really want to find out what happened to this file or why someone did link wrong.

File: CHANGELOG.md [hunk 29]
```diff
@@ -1,50 +1,952 @@
...
+
+#### [0.4.1](https://github.com/shopsys/product-feed-zbozi/compare/v0.4.0...v0.4.1) - 2017-09-25
+##### Added
+- [CONTRIBUTING.md](https://github.com/shopsys/product-feed-zbozi/blob/master/CONTRIBUTING.md) 
+- [template for github pull requests](https://github.com/shopsys/product-feed-zbozi/blob/master/docs/PULL_REQUEST_TEMPLATE.md) 
+##### Changed
+- Dependency [shopsys/product-feed-interface] upgraded from ~0.3.0 to ~0.4.0 
+- [`ZboziFeedConfig`](https://github.com/shopsys/product-feed-zbozi/blob/master/src/ZboziFeedConfig.php) now filters not sellable products 
+- [`ZboziFeedConfig`](https://github.com/shopsys/product-feed-zbozi/blob/master/src/ZboziFeedConfig.php) implemented method `getAdditionalData()` 
+- [`TestStandardFeedItem`](https://github.com/shopsys/product-feed-zbozi/blob/master/tests/TestStandardFeedItem.php) implemented method `getCurrencyCode()`
```
[the remark this answers is not in the document]
Comment (MattCzerner) [reply CHANGELOG.md]: same as heureka feed.

File: CHANGELOG.md [hunk 30]
```diff
@@ -1,50 +1,952 @@
...
+#### [0.4.0](https://github.com/shopsys/product-feed-zbozi/compare/v0.3.0...v0.4.0) - 2017-09-12
+##### Added
+- New dependencies for dev 
+    - phpunit/phpunit >=5.0.0,<6.0
+    - twig/twig 1.34.0
+    - twig/extensions 1.3.0
+- New automatic test that is controlling right behaviour of plugin 
+- Added travis build icon into [README.md](https://github.com/shopsys/product-feed-zbozi/blob/master/README.md) 
+##### Changed
+- Dependency [product-feed-interface](https://github.com/shopsys/product-feed-zbozi/blob/master/shopsys/product-feed-interface) upgraded from ~0.2.0 to ~0.3.0
```
[the remark this answers is not in the document]
Comment (MattCzerner) [reply CHANGELOG.md]: same as heureka feed.

File: CHANGELOG.md (hunk 17 above)
Comment (MattCzerner) [reply CHANGELOG.md]: Old file that does not exist anymore

File: CHANGELOG.md (hunk 25 above)
Comment (MattCzerner) [reply CHANGELOG.md]: this acutally does not exist anymore, i made change so it actually looks into the coding-standards package, but it does not exist

File: CHANGELOG.md (hunk 26 above)
Comment (MattCzerner) [reply CHANGELOG.md]: same here

File: CHANGELOG.md (hunk 27 above)
Comment (MattCzerner) [reply CHANGELOG.md]: same here

**Comment by MattCzerner** [comment]
So Sváťa, that was really good job, i hope that we got everything correct now.

For your information, no packages shouldnt be in the alphabetical order, they are ordered by importance, which suggested @PetrHeinz , framework is always first since this is where will be all the important changes made. 

After discussion we had agreed on keeping UPGRADE.md since CHANGELOG.md started to look messy after I added upgrade informations.

File: CHANGELOG.md [hunk 31]
```diff
@@ -1,50 +1,935 @@
...
+    - they should be located in directory "Migrations" in the root of the bundle
+- command `shopsys:migrations:generate` now offers a list of available bundles for generating the migrations
+
+### [shopsys/plugin-interface]
+#### Removed
+- storing data via Data storage was removed
+    - removed interfaces `PluginDataStorageProviderInterface` and `DataStorageInterface`
+    - only Doctrine entities are further allowed for storing data
+
+#### [shopsys/shopsys]
```
Comment (simara-svatopluk) [inline CHANGELOG.md]: Please change this section to 3rd and Added section below to 4th

**Pull request merged**